psycop/common/sequence_models/tests/test_encoder_for_clf.py

```diff
@@ -56,11 +56,11 @@ def encoder_module() -> nn.Module:
 
 
 @pytest.fixture()
-def aggregation_module() -> AveragePooler:
+def aggregation_module() -> CLSAggregationModule:
     """
-    A mean pooling module
+    Aggregation module that takes the hidden state of the first token (i.e. the CLS token)
     """
-    return AveragePooler()
+    return CLSAggregationModule()
```

Comment:
I would test both aggregation modules

Reply:
you can do this using pytest.mark.parametrize.

Reply:
very cool, didnt know that